### PR TITLE
Hocs 2619 serenity automation drone 1.0

### PR DESCRIPTION
**Pull request by Paulcr-ho**
The reports are being saved in /drone/src/target/site/serenity. 
Modifying drone source of s3 bucket plugin.
Hardcoding bucket name.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -61,13 +61,13 @@ steps:
...
-      region: eu-west-2
-      secret_key: ${AWS_SECRET_ACCESS_KEY}
-      source: target/site/serenity/**/*
-      target: /reports/$${DRONE_BUILD_NUMBER}/
+    access_key:
+      from_secret: aws_access_key_id
+    bucket: cs-integration-testing-s3
+    region: eu-west-2
+    secret_key: ${AWS_SECRET_ACCESS_KEY}
+    source: /drone/src/target/site/serenity/**/*
```
Comment (zuzak) [inline .drone.yml:69]: Can we use an env var for this? Or maybe [hardcode it](https://docs.drone.io/pipeline/docker/syntax/workspace/)?

Comment (Paulcr-ho) [reply .drone.yml:69]: I have no idea. The documentation isn't great.

Comment (Paulcr-ho) [reply .drone.yml]: I would prefer trying it as is to get a successful build.

**Review by LiamMacP — APPROVED** [review]

File: .drone.yml [hunk 2]
```diff
@@ -61,19 +61,14 @@ steps:
...
-    AWS_ACCESS_KEY_ID:
+    access_key:
       from_secret: aws_access_key_id
-    AWS_SECRET_ACCESS_KEY:
+    secret_key:
       from_secret: aws_secret_access_key
-    S3_BUCKET_NAME:
-      from_secret: s3_bucket_name
+    source: /drone/src/target/site/serenity/**/*
+    target: /reports/$${DRONE_BUILD_NUMBER}/
```
Comment (LiamMacP) [inline .drone.yml:71]: Will this report be places at the correct folder as the source has changed? and will that impact?

Comment (Paulcr-ho) [reply .drone.yml:71]: According to the logs from "execute-tests" the report is written to /drone/src/target/site/serenity

Comment (LiamMacP) [reply .drone.yml:71]: Not to worry then - looks good to me - think Douglas mentioned setting the workspace to `/drone/src`. But that can always be done later for me 👍🏼

Comment (Paulcr-ho) [reply .drone.yml:71]: I agree, we can neaten it up later. I just need to get it working first.

**Pull request merged**